MVVM Module/Source/Feed Presentation/FeedViewModel.swift

```diff
@@ -20,14 +20,23 @@ final class FeedViewModel {
 
 	var onLoadingStateChange: Observer<Bool>?
 	var onFeedLoad: Observer<[FeedImage]>?
+	var onFeedError: (() -> Void)?
+	var onHideFeedError: (() -> Void)?
 
 	func loadFeed() {
 		onLoadingStateChange?(true)
 		feedLoader.load { [weak self] result in
-			if let feed = try? result.get() {
+			switch result {
+			case let .success(feed):
 				self?.onFeedLoad?(feed)
+			case .failure:
+				self?.onFeedError?()
```

Comment:
To display an error on load failure you can call `onErrorStateChange?(Localized.Feed.loadError)`
```suggestion
				self?.onErrorStateChange?(Localized.Feed.loadError)
```